File: AndroidCourse/Java/Lesson_1/app/src/main/java/com/example/lesson_1/Student.java
package com.example.lesson_1;

public class Student {
    private final int id;
    private final String name;
    private final String surname;
    private final String grade;
    private final int birthdayYear;

    public Student(int id, String name, String surname, String grade, int birthdayYear) {
        this.id = id;
        this.name = name;
        this.surname = surname;
        this.grade = grade;
        this.birthdayYear = birthdayYear;
    }

    @Override
    public String toString() {
        return id +
                " " + name +
                " " + surname +
                " " + grade +
                " " + birthdayYear;
    }
}

File: AndroidCourse/Kotlin/Lesson_1/app/src/main/java/com/example/lesson_1/Student.kt
package com.example.lesson_1

class Student(
    private val id: Int,
    private val name: String,
    private val surname: String,
    private val grade: String,
    private val birthdayYear: Int
) {
    override fun toString(): String = "$id $name $surname $grade $birthdayYear"
}